text/0000-stable-simd.md

```diff
@@ -421,6 +425,9 @@ The current proposed mapping for x86 intrinsics is:
 | `short`         | `i16`     |
 | `int`           | `i32`     |
 | `long long`     | `i64`     |
+| `const int`     | `i32` [0] |
+
+[0] required to be compile-time constants.
```

Comment:
Ok so here I don't know if we can type-check this, but we should word this in such a way that allows us to make a breaking change in the future.

This might sound too heavy but if we were to write "if the argument is not a constant the behavior is undefined" that would allow us to generate bad code (e.g. a huge match statement) right now, and in the future, emit a compiler error or something.

I made this change via github, so I can't really edit it anymore AFAIK.